src/components/SearchBar.jsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react"
+import { getArticles, getTopics } from "../api"
+import Loading from "./Loading"
+
+function SearchBar({setArticlesList}) {
+ 
+  const [topicsList, setTopicsList] = useState()
+  const [topicQuery, setTopicQuery] = useState()
+  const [isLoading, setIsLoading] = useState(true)
+	const [error, setError] = useState(false)
+
+  useEffect(() => {
```

Comment:
We need to ensure error is set to false every time this useEffect runs

Reply:
I concur with Kev - before the API call in your useEffect, you should be setting error to false (even though it is initiated with false - that could change!!)